scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -489,72 +474,6 @@
 (gauges/defgauge [cook-mesos scheduler front-of-job-queue-mem] (fn [] @front-of-job-queue-mem-atom))
 (gauges/defgauge [cook-mesos scheduler front-of-job-queue-cpus] (fn [] @front-of-job-queue-cpus-atom))
 
-(defn ids-of-backfilled-instances
```

Comment:
Do these changes need to be backed by unit tests in `scheduler_test.clj` to ensure everything still works as expected?